client/components/theme-upgrade-modal/index.tsx

```diff
@@ -120,14 +120,12 @@ export const ThemeUpgradeModal = ( {
 
 		return {
 			header: (
-				<h1 className="theme-upgrade-modal__heading">
-					{ translate( 'Unlock this personal theme' ) }
-				</h1>
+				<h1 className="theme-upgrade-modal__heading">{ translate( 'Unlock this theme' ) }</h1>
```

Comment:
Nice that we're going with generic reference instead of explicit, will save us time if we ever change plan names/tiers again.